src/Commands/StopScan.ts

```diff
@@ -27,7 +27,8 @@ export class StopScan implements CommandModule {
             insecure: args.insecure as boolean,
             baseURL: args.api as string,
             apiKey: args.token as string,
-            proxyURL: (args.proxyBright ?? args.proxy) as string
+            proxyURL: (args.proxyBright ?? args.proxy) as string,
+            timeout: (args.timeout as number) * 1000
```

Comment:
Instead of repeatedly converting s to ms, you can use the `coerce` option

```ts
argv
.options({
  timeout: {
    coerce(arg: number) {
      return arg * 1000;
    },
    default: 30,
    type: 'number'
  }
})
```

Reply:
wow, cool! didn't know that, thanks! I'll amend the code now